fix gas forwarded in Shop level (#305)
OpenZeppelin/ethernaut: client/src/gamedata/en/descriptions/levels/shop_complete.md

> Contracts can manipulate data from other contracts the way they want.

Reviewer: ```suggestion
Contracts can manipulate data seen by other contracts in any way they want.
```
